clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1144,64 +1144,48 @@ protected String getConfigValue(ConfigKey key, String headerName) {
         }
     }
 
-    public String toHtmlTable() {
-        return toHtmlTable(Collections.<String, String>emptyMap());
+    public String toHtmlList() {
+        return toHtmlList(Collections.<String, String>emptyMap());
     }
 
-    private void addHeader(StringBuilder builder, String headerName) {
-        builder.append("<th>");
-        builder.append(headerName);
-        builder.append("</th>\n");
-    }
-
-    private void addColumnValue(StringBuilder builder, String value) {
-        builder.append("<td>");
-        builder.append(value);
-        builder.append("</td>");
-    }
-
-    /**
-     * Converts this config into an HTML table that can be embedded into docs.
-     * If <code>dynamicUpdateModes</code> is non-empty, a "Dynamic Update Mode" column
-     * will be included n the table with the value of the update mode. Default
-     * mode is "read-only".
-     * @param dynamicUpdateModes Config name -&gt; update mode mapping
-     */
-    public String toHtmlTable(Map<String, String> dynamicUpdateModes) {
+    public String toHtmlList(Map<String, String> dynamicUpdateModes) {
```

Comment:
Could we have a comment for this function?

Reply:
Yes, I re-added the initial comment slightly adapted